<launch>
<include file="$(find object_detect)/launch/depth_to_pointcloud.launch"/>
<node pkg="object_detect" type="object_detect_sync" name="object_detect_sync" output="screen">
	<param name="icp_max_iters" type="int" value="30"/>
	<param name="max_corr_dist" type="double" value="0.025"/>
	<param name="transform_eps" type="double" value="1e-8"/>
	<param name="voxex_grid_leaf_size" type="double" value="0.08"/>
	<param name="filt_feature_num" type="int" value="30" />
	<param name="min_feature_num" type="int" value="50" />
</node>
</launch>
